test/test.js

```diff
@@ -276,4 +279,11 @@ describe('HTTP Request testing', () => {
         assert.equal(res.turnByTurn.length, 1);
       });
   });
+  it('Get custom fields', () => {
```

Comment:
Do we want to include any tests for the post, update, or delete?

Reply:
Added a test for CREATE Custom Field.